ibis/backends/pyspark/compiler.py

```diff
@@ -1532,8 +1543,15 @@ def _get_interval_col(
             'Interval unit "{}" is not allowed. Allowed units are: '
             '{}'.format(dtype.unit, allowed_units)
         )
+
+    if isinstance(op.value, pd.Timedelta):
+        pandas_unit = _time_unit_mapping_pd_timedelta[dtype.unit]
+        value = getattr(op.value.components, pandas_unit)
+    else:
+        value = op.value
```

Comment:
What hits the else here? (What other than pd.Timedelta supported)?

Reply:
All of these existing tests for example hit the else: https://github.com/ibis-project/ibis/blob/master/ibis/backends/tests/test_temporal.py#L299
`ibis.interval`, `ibis.date`, etc.